src/Atmos/Model/turbulence.jl

```diff
@@ -3,7 +3,7 @@ using DocStringExtensions
 using CLIMA.PlanetParameters
 using CLIMA.SubgridScaleParameters
 export ConstantViscosityWithDivergence, SmagorinskyLilly, Vreman, AnisoMinDiss
-export turbulence_tensors
+export turbulence_tensors, symmetrize
```

Comment:
Neither of these should be exported as they are [not Atmos-specific](https://github.com/climate-machine/CLIMA/pull/687#discussion_r378580632) names.

Reply:
The Preset function has need of them, should they be imported instead ?

Reply:
reset this file to its original state: you can do `git checkout master src/Atmos/Model/turbulence.jl`

Reply:
@skandalaCLIMA can you fix this?

Reply:
This file has been reset to master. New Courant number reset functions do not use symmetrize.